v3/plugins/broadcommfd/cobol-language-support/0.9.1/meta.yaml

```diff
@@ -11,7 +11,7 @@ repository: https://github.com/eclipse/che-che4z-lsp-for-cobol
 category: Language
 spec:
   containers:
-    - image: eclipse/che-remote-plugin-runner-java8:next
+    - image: eclipse/che-remote-plugin-runner-java8:7.6.0
```

Comment:
```suggestion
    - image: quay.io/eclipse/che-sidecar-java:8-0cfbacb
```

Reply:
it should be fixed in master branch as well

Reply:
Should be confirmed with @filipkroupa -- they target specific versions for compatibility AFAIK.

Reply:
@amisevsk As per your recommendation we removed the necessity of specific plugins version, so in this case we should be ok. But I don't know what is the difference between the two docker images. We always used `eclipse/che-remote-plugin-runner-java8` for Java 8 runtime. Should we move to the new docker image `quay.io/eclipse/che-sidecar-java`?